hello/srv/world.ts

```diff
@@ -1,7 +1,7 @@
-import type { Request } from "@sap/cds/apis/services"
```

Comment:
Looks like this file is replaced by the file with the same name in `hello/src/srv` folder.
Can this version be removed?